website/docs/how-does-it-work/issuance-status.md

```diff
@@ -60,7 +60,7 @@ At the moment, OpenAttestation only supports one DID method: `ethr`.
 
 ### Issuance
 
-DIDs [are significantly faster and incur not costs](/docs/verifiable-document/comparison). They could directly use the `targetHash` of the document (which is unique) and sign it using the private key associated. However for consistency with our initial design, we sign the `merkleRoot`.
+DIDs [are significantly faster and incur no costs](/docs/verifiable-document/comparison). They could directly use the `targetHash` of the document (which is unique) and sign it using the private key associated. However for consistency with our initial design, we sign the `merkleRoot`.
```

Comment:
Probably just a grammar typo. If I understood correctly, it meant that using DID will not incur costs until there is a need for revocation.